go/service/config.go

```diff
@@ -161,6 +161,38 @@ func (h ConfigHandler) SetUserConfig(_ context.Context, arg keybase1.SetUserConf
 	return nil
 }
 
+func (h ConfigHandler) SetPath(_ context.Context, arg keybase1.SetPathArg) error {
```

Comment:
Yes, that's true.

But a similar situation would happen if launchctl used the user's path and they changed the path after the service started.  It wouldn't be reflected.  Not exactly the same, but if someone is messing with their path, restarting the service seems like a reasonable thing to have to do as well.